app/views/games/show.html.erb

```diff
@@ -1,22 +1,18 @@
-<div id='chessboard'>
+<div class="col-xs-12 col-sm-8 col-sm-offset-2 col-md-6 col-md-offset-3 no-padding">
 	<%= hidden_field :game_id, '', name: nil, id: nil, data: { game_id: @game.id } %>
```

Comment:
This change looks funny to me. Why not just add the Bootstrap screen size classes to the div#chessboard, and apply 0 padding to #chessboard? It's always looked strange to me when people use classes like 'no-padding'. Plus div#chessboard makes it clear what the wrapping container is.